crates/stages/src/stages/tx_lookup.rs

```diff
@@ -153,17 +153,34 @@ impl<DB: Database> Stage<DB> for TransactionLookupStage {
                         );
                     }
 
-                    if append_only {
-                        txhash_cursor.append(
-                            RawKey::<TxHash>::from_vec(hash),
-                            RawValue::<TxNumber>::from_vec(number),
-                        )?;
+                    let key = RawKey::<TxHash>::from_vec(hash);
+                    let res = if append_only {
+                        txhash_cursor.append(key, RawValue::<TxNumber>::from_vec(number))
                     } else {
-                        txhash_cursor.insert(
-                            RawKey::<TxHash>::from_vec(hash),
-                            RawValue::<TxNumber>::from_vec(number),
-                        )?;
+                        txhash_cursor.insert(key, RawValue::<TxNumber>::from_vec(number))
+                    };
+
+                    #[cfg(feature = "optimism")]
+                    // below bedrock, transaction nonces can be replayed.
+                    //
+                    // for importing chain segment up until bedrock, end block may be bedrock block.
```

Comment:
what's the impact of this?

does this skip txs? if so can we instead filter those txs out during import?

Reply:
filter them out based on what if not reading db, like it does here when it tries to insert?

Reply:
this only concerns tx duplicates?
there shouldn't be too many of those
should be possible to identify those

Reply:
you mean hardcode the transactions?

Reply:
hardcoded will work but need to scrape logs first, blocking this pr